app/webpacker/css/admin_v3/components/pagination.scss

```diff
@@ -14,23 +18,42 @@
     background-color: $color-1;
     @include defaultBoxShadow;
     border-radius: 4px;
-    color: $color-9;
+    color: $color-8;
+    font-weight: 600;
+
+    &.current,
+    &.gap {
+      cursor: default;
+    }
```

Comment:
As the `.cursor` and `.gap` elements are only `span`s, I don't think this rule is needed. Maybe something for the next PR :)